responses/de/HassRespond.yaml

```diff
@@ -0,0 +1,9 @@
+language: de
+responses:
+  intents:
+    HassRespond:
+      hello: "Hallo vom Home Assistant."
+      listening: "Nein, ich höre nur zu, wenn du das Signalwort sprichst."
+      data: "Deine Daten liegen auf deinem Home Assistant server."
+      commands: "Um zu erfahren, was du alles fragen oder sagen kannst, besuche Home Minus Assistant Punkt I.O. Slash voice."
```

Comment:
I'm not sure about this spelling of the url :thinking:

Reply:
Agree, for now we keep the aim at voice though. Maybe a future iteration could be that we handle actually URLs better? 🤷